lib/realtime/postgres_cdc.ex

```diff
@@ -37,14 +45,7 @@ defmodule Realtime.PostgresCdc do
   end
 
   def filter_settings(key, extensions) do
-    [cdc] =
-      Enum.filter(extensions, fn e ->
-        if e.type == key do
-          true
-        else
-          false
-        end
-      end)
+    [cdc] = Enum.filter(extensions, fn e -> if e.type == key, do: true, else: false end)
```

Comment:
I think this can be further simplified with:

```suggestion
    [cdc] = Enum.filter(extensions, fn e -> e.type == key end)
```

Reply:
🤦 yeah it does 😆